app/views/components/link.scala.html

```diff
@@ -0,0 +1,24 @@
+@*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@this()
+
+@(text: String, textHidden: Option[String] = None, call: Call, target: String = "_self", id: Option[String] = None, classes: String = "govuk-link govuk-link--no-visited-state")
+
+@hasHint = @{textHidden.exists(_.nonEmpty)}
+
+<a class="@classes" href="@call" target=@target @id.map{id => id="@id"}
+ ><span @if(hasHint){aria-hidden="true"}>@text</span>@if(hasHint){<span class="govuk-visually-hidden">@{textHidden.getOrElse("")}</span>}</a>
```

Comment:
Guess it was decided to not create components manually as suggested by @TheNickWilson . Is it still valid @liandra-giacomine

Reply:
think the only reason is to not to duplicate the feedback section content and links multiple places and have common feedback section that will be used across all successful journey pages